### DISCUSSION TITLE
how to

### DISCUSSION THREAD

**Discussion by F-Fichter** [Fishwaldo/sophgo-sg200x-debian#22] [General]
hello, 

how i can  load up ardunio/freertos images on the small C906 core.  ?

have you a link to make this ?

thk

**Comment by Fishwaldo** [comment]
Copy the elf file to /lib/firmware/ and then echo the file name to /sys/class/remoteproc/remoteproc0/filename. Then echo "start" or "stop" to the state file in that same directory. 

MilkV didn't release the source to the program they wrote for the buildroot so it's a manual process here.

**Reply by BoryaGames** [reply]
We got the [source](https://github.com/kubuds/sophgo-arduino/tree/sg200x-dev/tools/burn) for their `burnd` since November 2024, btw.

**Comment by F-Fichter** [comment]
hello,

 You say  elf file, it's my script ?  or is the arduino or freertos system ?
have you manual for lichee rv nano, i don't find info about this system 

thk

**Comment by Fishwaldo** [comment]
its the output from the compiler after you compile your Ardunio application.
